keymapper/injection/macros/macro.py

```diff
@@ -287,7 +287,7 @@ async def run(self, handler):
                 if asyncio.iscoroutine(coroutine):
                     await coroutine
             except Exception as e:
-                logger.error(f'Macro "%s" failed: %s', self.code, e)
+                logger.error(f'Macro "{self.code}" failed: {e}')
```

Comment:
lazy logging is considered good style by pylint for performance reasons. Also see https://docs.python.org/3.8/howto/logging.html#optimization

Reply:
Well, except that logger.error will always do its thing and it doesn't apply here